.github/workflows/maven-verify.yml

```diff
@@ -25,3 +25,5 @@ jobs:
   build:
     name: Verify
     uses: apache/maven-gh-actions-shared/.github/workflows/maven-verify.yml@v3
+    with:
+      jdk-matrix: '[ "8", "11", "17", "21" ]'
```

Comment:
This shoudl be standalone change

Reply:
this change is required to show that with Java 21 we have issue to run integration tests

Reply:
Sure it is. And I agree wiht it. But not in the way you did it.
You had mixed three parts in one PR I do not understand why. This should be separate PR, whcihshoudl go in before https://github.com/apache/maven-archetypes/pull/20

Aslo you are reverting the change which was agreed during review of #18